example_governance_doc.md

```diff
@@ -43,17 +43,54 @@ These stakeholders comprise the Schema Working Group, or SWG.
+- [Effective graph stewardship](https://www.apollographql.com/tutorials/effective-stewardship)
+- [Observability in GraphOS](https://www.apollographql.com/tutorials/supergraph-observability)
+- [Testing (unit and load testing on both the client and the server)](https://www.apollographql.com/tutorials/testing)
+
+### Additional Resources:
+- A great reference for GraphQL is the official [GraphQL website](https://graphql.org/learn/)
+- Teams are also encouraged to run through [Supergraph Architecture Framework (SAF) assessment](https://saf.apollographql.com/) to better understand their GraphQL posture across several pillars like Reliability, Security, Operational Excellence, Performance and Developer Experience.
+
 ## Background
 
 As ACME has grown over the past year, we realized that our existing REST API solution was failing to scale as we began to spin up new client applications and data requirements quickly changed for each new addition. 
 
 As a result, we decided on using GraphQL to power our new API in order to quickly add and change data as requirements changed. Additionally, GraphQL offered us an opportunity to enable clients to easily access the data they cared about. 
 
-In order to accelerate development speed, we landed on using [Apollo's Federation](https://www.apollographql.com/docs/federation) specification which enables portions of the overall graph to be known as "subgraphs." The overall graph is known as the supergraph. 
+In order to accelerate development speed, we landed on using [Apollo's Federation](https://www.apollographql.com/docs/federation) specification which enables portions of the overall graph to be known as "subgraphs." The overall graph is known as the supergraph. These principles are outlined in our Mission and Vision statement below:
+
+### Mission and Vision
+
+ACME is looking to establish a supergraph that is the preferred development platform for all services.
+
+The goal of the ACME supergraph is to be the aggregation layer on top of the a domain-based services architecture, defining and mapping relationships in the data across bounded contexts. Key to our success will be to make the supergraph the preferred mode of development over REST APIs as measured by the percentage of requests going to our GraphQL instance vs their REST counterparts. 
+
+The surface of the graph will span each of our current domain services, beginning with the most used domains first.
+
+The success outcomes for the mission established by ACME are as follows:
+
+1. We will simplify the API landscape by making the GraphQL supergraph the preferred platform over continued development of RESTful APIs. 
+2. Our focus will be on building reusable Domain APIs to focus on reusability, maintainability and scalability while minimizing the timelines required to update business logic.
+3. We will also establish a "GraphQL Shepherds" guild to play a key role in training and up-leveling GraphQL expertise across the dev teams.
```

Comment:
Don't we typically try to shy away from "guilds" here?

Reply:
Fair - I think I'll just change this to a more generic term like "group"